### PR TITLE
Issue #551 Revise Readme to be more accessible

### PR DESCRIPTION
**Pull request by bcgoss**
I have added some language to README.md which should make the document more accessible to people who are not familiar with Faraday. I am happy to make further adjustments based on your feedback. Thanks for taking a look!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,6 +23,23 @@ Available at [rubydoc.info](http://www.rubydoc.info/gems/faraday).
...
+response = Faraday.get 'http://sushi.com/nigiri/sake.json'
+```
+A simple `get` request can be performed by using the syntax described above. This works if you don't need to set up anything; you can roll with just the default middleware
+stack and default adapter (see [Faraday::RackBuilder#initialize](https://github.com/lostisland/faraday/blob/master/lib/faraday/rack_builder.rb)).
+
+A more flexible way to use Faraday is to start with a Connection object. If you want to keep the same defaults, you can use this syntax:
+
+```ruby
+conn = Faraday.new(:url => 'http://www.example.com') 
+resposne = conn.get '/users'                 # GET http://www.example.com/users'
```
Comment (iMacTia) [inline README.md]: small typo

File: README.md [hunk 2]
```diff
@@ -23,6 +23,23 @@ Available at [rubydoc.info](http://www.rubydoc.info/gems/faraday).
...
+stack and default adapter (see [Faraday::RackBuilder#initialize](https://github.com/lostisland/faraday/blob/master/lib/faraday/rack_builder.rb)).
+
+A more flexible way to use Faraday is to start with a Connection object. If you want to keep the same defaults, you can use this syntax:
+
+```ruby
+conn = Faraday.new(:url => 'http://www.example.com') 
+resposne = conn.get '/users'                 # GET http://www.example.com/users' 
+```
+
+Connections can also take an options hash as a parameter or be configured by using a block. Checkout the section called Advanced middleware usage for more details about how to use this block for configurations.
```
Comment (iMacTia) [inline README.md]: Can you please insert an anchor link to the Advanced middleware usage section?

**Comment by bcgoss** [comment]
Ok, I've made the suggested change. Let me know if you have any more, or other documentation you want another person looking at.

**Comment by iMacTia** [comment]
Looks good to me 👍 
Thanks

**Pull request merged**